extensions/machine-learning-services/src/modelManagement/deployedModelService.ts

```diff
@@ -92,16 +104,40 @@ export class RegisteredModelService {
 			}
 		}
 	}
+	private loadModelData(row: azdata.DbCellValue[]): RegisteredModel {
+		return {
+			id: +row[0].displayValue,
```

Comment:
Is the `+` intentional?  (learning moment for me: if so, what does it do?)

Reply:
it convert to number

---
In reply to: [397442655](https://github.com/microsoft/azuredatastudio/pull/9609#discussion_r397442655) [](ancestors = 397442655)